src/Nethermind/Nethermind.Serialization.Rlp/AccountDecoder.cs

```diff
@@ -253,5 +253,52 @@ private Hash256 DecodeCodeHash(ref Rlp.ValueDecoderContext rlpStream)
 
             return codeHash;
         }
+
+        private ValueHash256 DecodeStorageRootStruct(ref Rlp.ValueDecoderContext rlpStream)
+        {
+            ValueHash256 storageRoot;
+            if (_slimFormat && rlpStream.IsNextItemEmptyArray())
+            {
+                rlpStream.ReadByte();
+                storageRoot = Keccak.EmptyTreeHash.ValueHash256;
+            }
+            else
+            {
+                storageRoot = rlpStream.DecodeValueKeccak()!.Value;
```

Comment:
if for some reason DecodeValueKeccak() returns null, the underlaying value will probably be a zero-value struct(?)